github-orgs/kubeflow/org.yaml

```diff
@@ -1074,6 +1074,15 @@ orgs:
             privacy: closed
             repos:
               pipelines: write
+          kubeflow-outreach-committee:
+            description: Kubeflow Outreach Committee
+            members:
+            - chasecadet
+            - varodrig
+            privacy: closed
+            repos:
+              community: write
+              website: write
```

Comment:
Why do you need write access to those repos ?

Reply:
discussed with Andrey, we do not need to give access to this group. we'll be providing access to each person assigned to the team.